sdk/storage/azure-storage-file-share/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+- `ShareFileClient.downloadToFile()` now retries for a maximum of 5 times when the download fails due to a network error.
```

Comment:
Let's also include what the previous maximum (15) and note both that the number of reties was unintentional and that the additional retires past the intended maximum of 5 could lead to writing in incorrect positions in the file.